applications/luci-app-uhttpd/htdocs/luci-static/resources/view/uhttpd/uhttpd.js

```diff
@@ -0,0 +1,263 @@
+'use	strict';
+'require	view';
+'require	form';
+'require	fs';
+'require	uci';
+
+
+return	view.extend({
+	load:	function	()	{
+	return	Promise.all([
+	uci.load('uhttpd'),
+	]);
+	},
+	render:	function()	{
+	var	lhttp	=	null;
+	var	lhttps	=	null;
+	var	cert_file	=	null;
+	var	key_file	=	null;
+	var	ucs	=	null;
+	var	uhttpdMap	=	new	form.Map('uhttpd',	_('uHTTPd'),
+	_('A	lightweight	single-threaded	HTTP(S)	server'));
+	
+	ucs	=	uhttpdMap.section(form.TypedSection,	'uhttpd');
+	ucs.addremove	=	true;
+	ucs.anonymous	=	false;
+
+	ucs.tab("general",	_("General	Settings"));
+	ucs.tab("server",	_("Full	Web	Server	Settings"),	_("For	settings	primarily	geared	to	serving	more	than	the	web	UI"));
+	ucs.tab("advanced",	_("Advanced	Settings"),	_("Settings	which	are	either	rarely	needed	or	which	affect	serving	the	WebUI"));
```

Comment:
I think you have gone too far with the change of spaces to tabs. You only needed to do it for the "indentation", but not for the rest of the code. The spaces in the rest of the code, like between the strings, or like separator for parameters in the functions, must remain with spaces.

Reply:
Okay! I'll fix it very soon, I thought to change all spaces to tabs because of consistency.

Reply:
I fixed the issue

Reply:
I recommend to execute some automatic formatter. You can configure it with tabs for formatting. There are lots of indentations wrong.
You can always give a second adjust by hand.

Reply:
Sure thing! I formatted it properly.